### DISCUSSION TITLE
Consider publishing docs with a static site generator

### DISCUSSION THREAD

**Discussion by onedr0p** [onedr0p/cluster-template#1239] [Ideas]
One problem with this is when the user creates a repo from the template the docs and other files will carry over to the template.

There is no supported way to ignore files on generated repos from templates
https://github.com/orgs/community/discussions/22183

**Comment by onedr0p** [comment]
@alex-matthews I have not forgotten about this. There are some difficulties here since I would like everything contained to this repo. It's unfortunate that GitHub doesn't let us ignore certain files when someone creates a template.

**Comment by alex-matthews** [comment]
Could #1124 accomplish this?

**Comment by onedr0p** [comment]
Not really because there's no way to have it automatically run, it's up to the person who created the template to run that command manually.

**Comment by onedr0p** [comment]
Looks like enabling github wiki here is a no go, there's no way for people to PR changes to it.

https://stackoverflow.com/questions/10642928/how-can-i-make-a-pull-request-for-a-wiki-page-on-github

**Comment by onedr0p** [comment]
@alex-matthews I think we can move forward with mdbook in a docs folder if you still wanted to help document this. Lots have changed recently and the readme is getting a bit 🍝
